.husky/pre-commit

```diff
@@ -11,3 +11,7 @@ printf "No type errors! 🎉🎉🎉\n"
 printf "Linting project...\n"
 yarn workspaces foreach -p run lint
 printf "No lint errors! 🎉🎉🎉\n"
+
+printf "E2E testing...\n"
+yarn workspace @app/client run test:e2e:ci
+printf "All tests passing! 🎉🎉🎉\n"
```

Comment:
How fast do these run btw?  If it's like 10 minutes, may be more worthwhile to run on CI/CD rather than making the dev wait for e2e tests to pass before commiting.

Reply:
Well it's relatively quick on my machine (like 30 seconds or so), but that's just one test and it may be slower on other's machines.

Reply:
I think it's okay for now to leave it in tbh, but your call.  We can always add or remove it later.